localrpc/translateAobjsFwd.go

```diff
@@ -7,6 +7,16 @@ import (
 	to "github.com/MadBase/MadNet/proto"
 )
 
+// TODO: ForwardTranslateTXOut and ReverseTranslateTXOut need to be updated
```

Comment:
This can be solved by generating the associated protobuf definitions in proto and importing the object instead of using the objects from /application. Please open a ticket for this item. Kasper will pop this item as he is familiar with the proto system from previous work. This will not block PR at this time.